modules/apps/portal-workflow/portal-workflow-task-web/src/main/resources/META-INF/resources/edit_workflow_task.jsp

```diff
@@ -196,6 +196,7 @@ renderResponse.setTitle(workflowTaskDisplayContext.getHeaderTitle(workflowTask))
 										linkCssClass="btn btn-monospaced btn-outline-secondary"
 										markupView="lexicon"
 										message="view[action]"
+										target="_blank"
```

Comment:
I read a little bit more about adding this, and I found this: "Just make sure not to use them on your own website pages when you use internal links. Doing so could cause problems in your own site analytics and with Googlebot crawling and indexing your site structure." (see https://www.elegantthemes.com/blog/wordpress/rel-noopener-noreferrer-nofollow)

I understand that this will always be internal links no? 

Regards.

Reply:
ah good point, I had forgotten about that. Yeah you are right, this would be internal links so we don't have to worry about it